packages/rancher-logging/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -22,3 +22,11 @@
 +  {{- end }}
 +  {{- end }}
 +{{- end }}
++
++{{- define "windowsPathPrefix" }}
++{{- $pathPrefix := (required "Must provide value for .Values.global.cattle.rkeWindowsPathPrefix" .Values.global.cattle.rkeWindowsPathPrefix) -}}
++{{- if (contains "//" $pathPrefix) -}}
++{{- fail ".Values.global.cattle.rkeWindowsPathPrefix must not contain forward-slashes" -}}
++{{- end -}}
++{{- dir .Values.global.cattle.rkeWindowsPathPrefix }}
```

Comment:
Not sure why we are using [`dir`](https://helm.sh/docs/chart_template_guide/function_list/#dir) since it would return one level up from what the pathPrefix defined is (e.g. `\opt\rke` would return `\opt`)